shared/constants/teams.js

```diff
@@ -175,11 +175,8 @@ const userIsActiveInTeamHelper = (
   return member && member.active
 }
 
-const getConvIdsFromTeamName = (state: TypedState, teamname: string): I.Set<ChatTypes.ConversationIDKey> =>
-  I.Set(getTeamChannelInfos(state, teamname).keys())
-
-const getBadgeSubscribe = (state: TypedState, teamname: string): boolean =>
-  !state.teams.chosenChannelsForTeam.has(teamname)
+const isTeamWithChosenChannels = (state: TypedState, teamname: string): boolean =>
+  !state.teams.teamsWithChosenChannels.has(teamname)
```

Comment:
This shouldn't be negated I think

Reply:
D'oh, fixed! Pining for automated integration tests...